maildir2gmail.py

```diff
@@ -98,10 +98,10 @@ def mark_appended(self, filename):
 
     @property
     def database(self):
-        import dbhash
         if self.__database is None:
-            dbname = os.path.abspath(os.path.splitext(sys.argv[0])[0] + '.db')
-            self.__database = dbhash.open(dbname, 'w')
+            import shelve
+            dbname = os.path.abspath(os.path.splitext(sys.argv[0])[0])
+            self.__database = shelve.open(dbname)
```

Comment:
Wondering if it makes sense to hold a copy in memory, with writeback=True. After all, this script will access many values in a very short timeframe.

Reply:
In practice, i was migrating a bunch of Courier style Maildirs with thousands of messages each and not using the writeback option. While you may be right (thousands of emails is a lot in human scale), the performance was very acceptable without using this option. If you feel strongly about it i'll add it, but in terms of usability I did not think "gee, this shelve library is slow."

I am all migrated now and don't feel like the performance needs to be better, What do you think?